iOS/Scripts/Package.swift

```diff
@@ -0,0 +1,39 @@
+// swift-tools-version:4.0
+import PackageDescription
+
+let mainAppName = "Scripty"
+let frameworkName = "ScriptyCore"
+
+let package = Package(
+    name: mainAppName,
+    products: [
+      .executable(
+        name: mainAppName,
+        targets: [mainAppName]
+      ),
+      .library(
+        name: frameworkName,
+        targets: [frameworkName]
+      )
+    ],
+    dependencies: [
+        .package(
+          url: "https://github.com/JohnSundell/Files.git",
+          from: "2.0.0"
```

Comment:
looks different from what was generated in `Package.resolved` 🤔

Reply:
Yeah-  i'm not entirely sure how SPM resolves this - i'll have to do some looking into it. My recollection is that somewhere you can specify major/minor/patch to pin to but this might be the min version. I'll take a look when I get in

Reply:
Aha: According to [this description](https://github.com/apple/swift-package-manager/blob/master/Documentation/PackageDescriptionV4.md#dependencies) I need to use something like `.upToNextMinor(2.0.0)` in order to get the same behavior as Cocoapods gives you for `~> 2.0.0`.